plugins/SLURM-Integration/ufm_slurm_base.py

```diff
@@ -48,7 +48,7 @@ def init(self):
         self.app_resources_limit = self.general_utils.get_conf_parameter_value(Constants.CONF_APP_RESOURCES_LIMIT)
         if self.app_resources_limit and int(self.app_resources_limit) < -1:
             logging.error("app_resources_limit param must be an integer number greater than -1!")
-            sys.exit(1)
+            sys.exit(self.should_fail)
```

Comment:
why to fail prolog and epilog because of wrong configuration value?
in case wrong configuration value was set (-2) set the default value and notify the user with message:
logging.error("app_resources_limit param must be an integer number greater than -1, (got $config_resource_limit), using default value - 1")

Then no need to exit the script for that and in that case we can remove the "should_fail_slurm_job" from configuration + remove all related logic

Reply:
I fixed the logging and not exiting the code in case of invalid value, regarding the removal of should_fail_slurm_job we can't remove it because it's used in many pleases.